docs/client/AccountInvitationsApi.md

```diff
@@ -117,32 +131,38 @@ Name | Type | Description  | Notes
 
 ### HTTP request headers
 
- - **Content-Type**: Not defined
- - **Accept**: application/json
+- **Content-Type**: Not defined
+- **Accept**: application/json
 
-[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
+[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints)
+[[Back to Model list]](../README.md#documentation-for-models)
+[[Back to README]](../README.md)
 
-# **InviteAccount**
-> AccountInvite InviteAccount(ctx, accountId, optional)
-Invite account member
 
+## InviteAccount
 
+> AccountInvite InviteAccount(ctx, accountId, optional)
+
+Invite account member
 
 ### Required Parameters
 
+
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
-  **accountId** | **string**|  | 
+**ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+**accountId** | **string**|  | 
  **optional** | ***InviteAccountOpts** | optional parameters | nil if no parameters
 
 ### Optional Parameters
+
 Optional parameters are passed through a pointer to a InviteAccountOpts struct
 
+
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
 
- **body** | [**optional.Interface of AccountInvite**](AccountInvite.md)|  | 
+ **accountInvite** | [**optional.Interface of AccountInvite**](AccountInvite.md)|  | 
```

Comment:
body -> accountInvite seems like a fix, but just want to make sure it is intentional.